internals/overlord/checkstate/handlers.go

```diff
@@ -91,9 +91,8 @@ func (m *CheckManager) doPerformCheck(task *state.Task, tomb *tombpkg.Tomb) erro
 		}
 
 		m.incSuccessCount(config)
+		m.updateCheckData(config, changeID, CheckStatusUp, 0)
```

Comment:
This had to be moved out of the if statement. That was okay before as the status was already up, so it wouldn't change in this case, but now it's changing from "unknown" to "up" so needs to be updated.